src/python/frontend/options.py

```diff
@@ -95,10 +95,9 @@ def __init__(self):
         self._opts['logging']['level'] = logging.ERROR
         self._opts['logging']['file'] = None
 
-        self._opts['dbhost'] = None
-        # Change the default here to not update the metadiags database on the Classic Viewer server
-        self._opts['dbopts'] = 'no'
+        self._opts['dbhost'] = "https://diags-viewer.llnl.gov"
```

Comment:
Yeah, I'll be bringing that online after working with CSP to make sure we're all hunky-dory.